src/uniprotkb/components/entry/GoRibbon.tsx

```diff
@@ -255,8 +255,7 @@ const GoRibbon = ({
   );
 
   return (
-    <div className="GoRibbon">
-      <h3 data-article-id="gene_ontology">GO annotations</h3>
+    <div className="GoRibbon" data-article-id="gene_ontology">
```

Comment:
I think this is this here causing the extra "i" below the whole section, as the "i" will be added inline after the content, and here the content is a block section containing the whole go-ribbon section.
We'd need to find somewhere else to attach it to, that would be inline and not already clickable (so maybe not the tab label itself). Maybe a bit of extra text at the top of the section?

Reply:
I followed your suggestion added a bit of a preamble to both of the GO tabs and attached the `data-article-id` to this.